lib/steep/cli.rb

```diff
@@ -20,6 +22,8 @@ def self.available_commands
     end
 
     def setup_global_options
+      OptionParser.new.parse(argv)
```

Comment:
I think we don't need to `parse` here. Right?

Reply:
Do you think this way is correct?

```diff
     def setup_global_options
-      OptionParser.new.parse(argv)
+      version = OptionParser.new.version
+      if version
+        stdout.puts version
+        return false
+      end
 
       true
     end
```

This way outputs the only version without the program name (same as `steep version` subcommand).

```sh
# Before
$ ./exe/steep --version
steep 0.7.0

# After
$ ./exe/steep --version
0.7.0
```

Reply:
@soutaro ping?

Reply:
I know the difference, but don't think we should take care of that now. (I'll do that later.)